CapstoneProject/Models/MoodDetection.m

```diff
@@ -0,0 +1,82 @@
+//
+//  MoodDetection.m
+//  CapstoneProject
+//
+//  Created by Apoorva Chilukuri on 7/15/22.
+//
+
+#import "MoodDetection.h"
+
+@interface MoodDetection ()
+
+@property (strong, nonatomic) NSMutableArray *userTextWords;
+
+@end
+
+@implementation MoodDetection
+
+- (void) initializer{
```

Comment:
Instead of creating a new method, you can override the init function. When creating, the function will be called.

```
- (instancetype)init {
    if (self = [super init]) {
        // ... your initializer 
    }
    
    return self;
}
```